app/api/salesforce/conversations/route.ts

```diff
@@ -14,6 +15,13 @@ import {
 } from "@/app/api/salesforce/utils";
 import { HANDOFF_AUTH_HEADER } from "@/app/constants/authentication";
 
+function containsChatRequestSuccess(messages: SalesforceChatMessage[]) {
+  return messages.some(
+    // TODO: Replace hardcoded message name
+    (message) => message.type === "ChatRequestSuccess",
```

Comment:
can we swap for the constant in PR 78?
```suggestion
    (message) => message.type === SALESFORCE_MESSAGE_TYPES.ChatRequestSuccess,
```

Reply:
Once they're merged up, yes.